apps/pie-docs/src/components/button/code/props.json

```diff
@@ -13,6 +13,30 @@
                 "item": ["medium"]
             }
         ],
+        [
+            "isResponsive",
+            {
+                "type": "code",
+                "item": ["true", "false"]
+            },
+            "If <b>true</b>, uses the next larger size on wide viewports.",
+            {
+                "type": "code",
+                "item": ["false"]
+            }
+        ],
+        [
+            "responsiveSize",
+            {
+                "type": "code",
+                "item": ["productive", "expressive"]
+            },
+            "Sets a specific size for xsmall button when rendered as responsive.",
+            {
+                "type": "code",
+                "item": ["productive"]
+            }
+        ],
```

Comment:
If you happen to update this PR again, I just noticed that a few lines below it says "buttom" not "button", otherwise we can fix it in a later PR